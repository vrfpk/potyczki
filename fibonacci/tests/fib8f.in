0000000065535000
