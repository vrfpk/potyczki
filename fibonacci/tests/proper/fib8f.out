17027187043151250
